postgres/postgres.go

```diff
@@ -62,6 +67,11 @@ func (s *Stmt) Prepare(db *sqlx.DB, SQL string) (statement *sql.Stmt, err error)
 	return
 }
 
+// Load postgres
+func Load() {
+	config.Adapter = Postgres{}
```

Comment:
please use a pointer of postgres struct as adapter implementation

Reply:
Is it to pass Postgres {} as a reference?
```go
config.Adapter = &Postgres{}
```
In the case already with the change of config.
```go
config.PrestConf.Adapter = &Postgres{}
```

Reply:
nice!!